src/MrdAuth0LaravelServiceProvider.php

```diff
@@ -31,10 +38,24 @@ public function register()
         // Load our config.
         $this->mergeConfigFrom(__DIR__.'/../config/mrd-auth0.php', 'mrd-auth0');
 
+
         // Bind the auth0 user repository implementation.
         $this->app->bind(Auth0UserRepository::class, function (Application $app) {
             $config = $app['config']['mrd-auth0'];
             return new Repository\Auth0UserRepository($config['model'], $config['jwt-model']);
         });
+
+        // Add a singleton for the Authentication SDK using laravel-auth0 config values to the service container.
+        $this->app->singleton(Authentication::class, function () {
+            $config = config('laravel-auth0');
+            return new Authentication(
+                $config['domain'],
+                $config['client_id'],
+                $config['client_secret'] ?? '',
```

Comment:
Ternary?

Reply:
It's actually PHP's [Null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)

Reply:
TIL